src/_border-styles.scss

```diff
@@ -0,0 +1,51 @@
+.bs-none {
+  border-style: none;
+}
+
+.bs-solid {
+  border-style: solid;
+}
+
+.bs-t-solid {
```

Comment:
Looking at these classes I think, we should go with you suggestion from the naming spreadsheet; defined the border style once for all edges and control where that border appears with other classes.

```
<div class="bt bb b-solid">...</div>
```

IMO we should avoid the idea that one element can apply multiple border styles

Reply:
ah okay - so it's just a matter of renaming `bs-l-solid` to `bl` etc?

so if I want a div with a full blue border I'd do `<div class="b-solid b-ocean bw-1">` and for a div with just a border on the top and bottom I'd do `<div class="bt bb b-ocean bw-1">`

Reply:
maybe `b-solid` should be `ba` then? I think that's what tachyons does.